addons/web_editor/static/src/js/editor/odoo-editor/src/commands/commands.js

```diff
@@ -518,10 +519,14 @@ export const editorCommands = {
             if (node.nodeType === Node.TEXT_NODE && !isVisibleStr(node) && closestElement(node).isContentEditable) {
                 node.remove();
             } else {
+                // Ensure nav-item lists are excluded from toggling
+                const isNavItemList = node => node.nodeName === 'LI' && node.classList.contains('nav-item');
                 let block = closestBlock(node);
-                if (!['OL', 'UL'].includes(block.tagName) && block.isContentEditable) {
-                    block = block.closest('li') || block;
-                    const ublock = block.closest('ol, ul');
+                block = isNavItemList(block) ? node : block;
+                if (!['OL', 'UL'].includes(block.tagName) && (block.isContentEditable || block.nodeType === Node.TEXT_NODE)) {
```

Comment:
@Zynton Conflicts Resolved!
Changes:
1. Had to tweak this :point_up:  `if` condition.
2. Added a commit to handle the case in `sanitize` that it should not remove a p tag inside a nav-item list.

Reply:
Oops I somehow had missed that you made it so that `block` could... not be a block. Anyway in that context this change makes sense and we'll keep it for this forward port. But it would be nice to make a quick PR to refactor this part.

Reply:
I am thinking about `node` but it is already used how about `nodeToToggle`?